examples/dbt_dagster_tutorial/tutorial_template/jaffle_shop/models/staging/stg_orders.sql

```diff
@@ -0,0 +1,26 @@
+## Refer to Using dbt with Dagster, part one for info about this file:
+## https://docs.dagster.io/integrations/dbt/using-dbt-with-dagster/part-one
+
+with source as (
```

Comment:
is this the version of the model you would get from the jaffle shop repo? it look pretty different from how my dbt model ended up looking when i went through the tutorial, and seeing all this stuff about seeds had me confused. It also felt kinda wrong to replace such a long file with the relatively short code block from the tutorial. For reference, when i went through the tutorial i ended up with a file that looked like this for customers

```
select
    id as customer_id,
    first_name,
    last_name
from {{ ref(customers_raw)}}

```
or something similar to that. basically the only difference was ref instead of source

Reply:
went back to the dbt tutorial to find the snippet i ended up with - it looks like this 

```
select
    id as customer_id,
    first_name,
    last_name

from raw.jaffle_shop.customers
```